src/interfaces/chain.cpp

```diff
@@ -177,6 +198,56 @@ class ChainImpl : public Chain
         LOCK(cs_main);
         return GuessVerificationProgress(Params().TxData(), LookupBlockIndex(block_hash));
     }
+    void relayTransaction(const uint256& txid) override
```

Comment:
15288 why don't we have https://github.com/bitcoin/bitcoin/pull/15288/files#diff-efaf73429006d059e7fffab1bf2879769c90187f583d4857b3275b7bf79368d1R207-R211